docs/digital-object-full-view.md

```diff
@@ -0,0 +1,32 @@
+# View Modes: Digital Object Full View
+
+`View modes` tell Archipelago Digital Objects (ADOs) how to be displayed to the world. This documentation will give a brief overview of the View mode `Digital Object Full View` and how it can be used when creating or editing ADOs.
+
+## Digital Object Full View
+The `Digital Object Full View` is the default `View mode` used for the Media type `Photograph` and any other Media type that is not already configured to a View mode*. `Digital Object Full View` has the ability to zoom in, zoom out, full screen, rotate and display multiple images associated with an ADO.
+
+<details><summary>* Configuring ADO Types to View modes </summary>
+<div>
+
+The **ADO Type to View mode Mapping** page can be accessed at */admin/config/archipelago/viewmode_mapping*.
+This page will allow users to configure which `View mode` is used by default via the JSON key `type`.
```

Comment:
is used by default via the JSON key `type`. i don't have the wording right now, but i feel this needs to be more explicit. Like telling people (inverting the sentence?) that given a certain Media type (which is really any, like ANY JSON key `type` in your metadata) you can setup which existing view mode will be used. And give some example with JSON embeded or such. Do we have docs for this already? I feel i have this somewhere. Also **ADO Type to View mode Mapping** can/should be its own .MD really